server/reports/expiring-items/2_3_0/convert_data_js/src/utils.js

```diff
@@ -77,6 +80,9 @@ const calculateStockAtRisk = (
       }
     }
   }
+  if (stockAtRisk < 0) {
```

Comment:
I think expected usage should also be capped

Reply:
https://github.com/msupply-foundation/open-msupply/compare/5579-fix-remaining-report-bugs...5579-fix-remaining-report-bugs(Cap-expected-usage-to-totalNumberOfPacks)